app/views/admin/loans/timeline/_show_project_step.html.slim

```diff
@@ -0,0 +1,76 @@
+div.panel-body
+  div.step-details.row
+    div.col-lg-4.main-details.first-col
+      div.row
+        div.col-lg-1.check.first-col
+          input.select-step type="checkbox" class=(step.completed? ? 'completed-item' : 'incomplete-item') data-id== step.id
+        div.col-lg-11.first-col
+          h5.title
+            span.icon
+              - if step.milestone?
+                i.fa.fa-calendar-check-o
+              - else
+                i.fa.fa-flag
+            span.title-text = step.try(:summary)
+
+          p = step.try(:details)
+
+    div.col-lg-5.status-details
+      p.info.row
+        span.info-label.col-lg-3
+          = t("project_step.labels.status")
+
+        span.info-text.col-lg-9
+          - if step.admin_status
+            = t("project_step.status.#{step.admin_status}")
+          - else
+            = t("none")
+
+      p.info.row
+        span.info-label.col-lg-3
+          = t("project_step.labels.scheduled")
+
+        span.info-text.col-lg-9 style="background: #{step.scheduled_bg}"
+          = step.scheduled_date
+
+          - if step.completed?
```

Comment:
Why is this if statement more indented than the lines below. Does this do anything?